source/documentation/deploying_services/use_a_custom_domain.erb

```diff
@@ -1,32 +1,30 @@
 # Managing custom domains using the cdn-route service
 
-This section explains how to manage custom domains for your app with a content distribution network (CDN). The CDN:
+Apps on GOV.UK PaaS are given a domain of `APP_NAME.london.cloudapps.digital` by default. This is OK to use within your team and for development. However, you shouldn’t use it for a live, public service. This guide will help you to use a custom domain instead.
```

Comment:
I'm not very sure about the "you shouldn't use it for". It has the wrong tone, for my money. I think we should rephrase it to talk about how a live service will need to be accessed. Something like

> However, a live service will need likely need its own domain name, such as `my-service.service.gov.uk`

Reply:
Thanks Andy, I agree. I've pushed a commit which does this but I found it quite awkward to integrate. Re-review please?